.vscode/settings.json

```diff
@@ -4,12 +4,6 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": "explicit",
     "source.fixAll": "explicit"
-  },
-  "[typescript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-  },
-  "[typescriptreact]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
@wpdas i thought these entries were needed in order to format .tsx files on save?

Reply:
from @wpdas:

> the `fixAll` fixes the tsx files, based on prettier + eslint rules

Reply:
`fixAll` + prettier + eslint should fix all tsx, ts, js,... files